roles/system_packages/tasks/main.yml

```diff
@@ -34,6 +34,16 @@
   tags:
     - bootstrap_os
 
+- name: Install kernel-modules-extra on RHEL 10
+  package:
+    name: kernel-modules-extra-{{ ansible_kernel }}
+    state: present
+  when:
+    - ansible_os_family == 'RedHat'
+    - ansible_distribution_major_version in ['10']
+  tags:
+    - bootstrap_os
```

Comment:
Hi @tico88612,  
Why does RockyLinux 10 require installing kernel-modules-extra? Could you add some comments about this?

Reply:
I wrote in the commit, "Some of the kernel modules required by CNI are missing, installing
kernel-modules-extra can solve this problem."

The kernel modules like `ip_set` which required by `kube-proxy`.